openmetadata-ui/src/main/resources/ui/src/pages/tags/index.tsx

```diff
@@ -346,6 +369,20 @@ const TagsPage = () => {
     }
   };
 
+  const handleRenameSave = () => {
+    updateCategory({
+      name: (currentCategoryName || currentCategory?.name) ?? '',
+      description: currentCategory?.description ?? '',
```

Comment:
Why we are sending the description?

Reply:
this is the PUT call, we have to send all the required details!